pkgs/applications/misc/orca/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, pkgconfig, fetchurl, buildPythonApplication
+, autoreconfHook, wrapGAppsHook
+, intltool, yelp_tools, itstool, libxmlxx3
+, python, pygobject3, gtk3, gnome3
+, at_spi2_atk, at_spi2_core, pyatspi, dbus, dbus-python, pyxdg
+, louis ? null
+, xmodmap, xkbcomp, gsettings_desktop_schemas
+
+, speechdSupport ? true, speechd
+, brailleSupport ? false, brltty, brlapi ? null
+, setproctitleSupport ? true, setproctitle
+, gstreamerSupport ? true, gst_all_1, gst-python
+}:
+
+with lib;
+let
+  version = "3.26.0";
+  majorVersion = builtins.concatStringsSep "." (take 2 (splitString "." version));
+in buildPythonApplication rec {
+  name = "orca-${version}";
+  format = "other";
+  src = fetchurl {
+    url = "mirror://gnome/sources/orca/${majorVersion}/${name}.tar.xz";
+    sha256 = "0xk5k9cbswymma60nrfj00dl97wypx59c107fb1hwi75gm0i07a7";
+  };
+
+  # doCheck = true;
```

Comment:
This can be either removed or uncommented?

Reply:
Done